infra/prod/main.tf

```diff
@@ -49,3 +49,23 @@ module "function_app" {
 
   tags = local.tags
 }
+
+
+module "cms_function_app" {
+  source              = "../_modules/cms_function_app"
+  prefix              = local.prefix
+  env_short           = local.env_short
+  location            = local.location
+  location_short      = local.location_short
+  domain              = local.domain
+  resource_group_name = azurerm_resource_group.rg.name
+
+  virtual_network = {
+    name                = data.azurerm_virtual_network.itn_common.name
+    resource_group_name = data.azurerm_virtual_network.itn_common.resource_group_name
+  }
+  peps_snet_id                         = data.azurerm_subnet.private_endpoints_subnet.id
+  private_dns_zone_resource_group_name = data.azurerm_resource_group.weu-common.name
```

Comment:
The resource group where all private dns zone are stored is [io-p-rg-common](https://portal.azure.com/#@pagopait.onmicrosoft.com/resource/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-rg-common)

Reply:
Yes is the right rg, data referenced here is defined in https://github.com/pagopa/io-services-cms/blob/6c3f7e3f45d7d95e2faebed0ad6d4b2ed24bb97b/infra/prod/data.tf